common/lib/capa/capa/capa_problem.py

```diff
@@ -978,7 +974,7 @@ def response_a11y_data(self, response, inputfields, responsetype_id, problem_dat
             for description in description_tags:
                 descriptions[
                     "description_%s_%i" % (responsetype_id, description_id)
-                ] = description.text
+                ] = HTML(stringify_children(description))
```

Comment:
@robrap In CAPA instructors can add any kind of HTML, python and javascript and we're sure about the legacy problems that which tags they're already using. So for safe side, i think we can consider they'll use valid HTML.